crates/runtime/src/lib.rs

```diff
@@ -37,15 +39,17 @@ pub struct Runtime {
     pub app: Arc<app::App>,
     pub config: config::Config,
     pub df: Arc<DataFusion>,
+    pub pods_watcher: RefCell<podswatcher::PodsWatcher>,
```

Comment:
This doesn't need to be in a RefCell. Only `start_pods_watcher` needs it to be mutable, and that happens only on initialization. So we can pass a mutable reference to that function directly.

Reply:
Updated, thank you!